### DISCUSSION TITLE
Self hosted Supbase skips the sign in page

### DISCUSSION THREAD

**Discussion by mohamedFaris47** [supabase/supabase#13114] [Questions]
Greetings, I have deployed Supabase on a kubernetes cluster. Then when I go the supabase studio url the sign in page appears for less than a second then it redirects to the projects page directly. Is this the normal in this kind of deployment, and if so how get the sign in page back. I want to enable authentication with Azure AD, so please correct me I got something wrong.
Thank you

**Comment by Towerful** [comment]
The studio doesnt have any authentication for the self hosted version.  

https://supabase.com/docs/guides/self-hosting/docker#securing-the-dashboard

**Reply by sutidor** [reply]
Same question. What is the reason for the sign-in page?
I also noticed that the "authentication" tab in supabase studio looks different than in the videos (less options, no auth provider selection)

**Reply by Towerful** [reply]
I think it's because a lot of the cloud version doesn't actually use what the self hosted version uses.  
The realtime layer is a distributed cluster than managed multitenancy. For self hosted, a lot of that is turned off, or is forced to autogenerate a single tenant.  
They mentioned in a blog about logging that the self hosted logging doesn't capture as much data as cloud because it's not leveraging the same API gateway. Which means Kong is a self-hosted drop in that is functionally similar.

There are lots of parts of the code where there are `IS_PLATFORM` checks to switch various features on/off.   
The login page will be such a check where it just logs in as the default tenant.


A lot of the features are actually available, they are just a bit harder to get stuck into. The Auth providers are available as env vars in the supabase/gotrue repo, which you need to manually wire up in the docker-compose.
